### PR TITLE
[ANOMALY CLASSIFICATION] Add `sample.py` to `./external/anomaly_classification/tools/`

### PR DESCRIPTION
**Pull request by samet-akcay**
# Description
This PR adds `sample.py` to `./external/anomaly/anomaly_classification/tools/`. 

# BMM
- Build 🛠️ [#22601](https://ci.iotg.sclab.intel.com/job/IMPT/job/IMPTOps/22601/) successful. ✅

### CODE REVIEW

**Comment by samet-akcay** [comment]
run ote_sdk tests

**Comment by samet-akcay** [comment]
run ote_sdk tests

File: external/anomaly/anomaly_classification/tools/sample.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+from ote_sdk.entities.resultset import ResultSetEntity
+from ote_sdk.entities.subset import Subset
+from ote_sdk.entities.task_environment import TaskEnvironment
+from ote_sdk.entities.train_parameters import TrainParameters
+from ote_sdk.usecases.tasks.interfaces.evaluate_interface import IEvaluationTask
+from ote_sdk.usecases.tasks.interfaces.export_interface import ExportType
+from ote_sdk.usecases.tasks.interfaces.inference_interface import IInferenceTask
+from ote_sdk.usecases.tasks.interfaces.optimization_interface import OptimizationType
+
+logger = logging.getLogger("pytorch_lightning")
```
Comment (ashwinvaidya17) [inline external/anomaly/anomaly_classification/tools/sample.py]: Should we rename this to something like anomaly classification sample?

**Review by ashwinvaidya17 — DISMISSED** [review]
Clean! I have only one very minor comment.

File: external/anomaly/anomaly_classification/tools/sample.py (hunk 1 above)
Comment (samet-akcay) [reply external/anomaly/anomaly_classification/tools/sample.py]: yeah, would be nice. Here, I'm getting the pytorch lightning logger tough to append our stuff to pytorch lightning stuff

**Comment by djdameln** [comment]
Very nice implementation! Much cleaner than the test helpers we had previously. Will this be a replacement of those, or do we keep both?

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by samet-akcay** [comment]
run ote_sdk tests

**Pull request merged**